forge/src/main/java/org/openjdk/skara/forge/HostedRepositoryPool.java

```diff
@@ -117,7 +117,7 @@ private Repository cloneSeeded(Path path, boolean allowStale, boolean bare) thro
 
         private void removeOldClone(Path path, String reason) {
             if (Files.exists(path)) {
-                var preserved = path.resolveSibling(seed.getFileName().toString() + "-" + reason + "-" + UUID.randomUUID());
+                var preserved = path.resolveSibling(path.getFileName().toString() + "-" + reason + "-" + UUID.randomUUID());
```

Comment:
The filename here was based on the seed repo (which usually has a .git suffix) instead of the scratch repo that we are copying. That didn't really make sense.